<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kEasySkanSettings</class>
 <widget class="QWidget" name="kEasySkanSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>668</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="generalGB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QCheckBox" name="setPreviewDPI">
        <property name="text">
         <string>Set preview resolution (DPI)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="previewDPI">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>75</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>150</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>300</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">600</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QCheckBox" name="u_disableSelections">
        <property name="text">
         <string>Disable automatic selections</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QPushButton" name="revertOptions">
        <property name="text">
         <string>Revert scanner options to default values</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image saving</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="0" colspan="2">
       <widget class="QGroupBox" name="Identifiers">
        <property name="title">
         <string>Filename Identifiers</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QRadioButton" name="dateTimeAppend">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Appends a unique 'date and time' identifier to filename. e.g., MyImage-2018.12.31-23.59.59.jpg.&lt;/p&gt;&lt;p&gt;(format is always yyyy.MM.dd-hh.mm.ss and is independent of user's locale settings)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>&amp;Date and Time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="autoNumberAppend">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This scans the target directory for files named with the exact same prefix and automatically appends the right number.&lt;/p&gt;&lt;p&gt;Note that this overrides the other two options.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Au&amp;to Number</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="numberAppend">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This adds a number in a 4 digit format after &lt;span style=&quot; font-style:italic;&quot;&gt;Name Prefix,&lt;/span&gt; seperated by &amp;quot;-&amp;quot; e.g., My Image-0077.jpg.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Man&amp;ual Number</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QSpinBox" name="fileNumber"/>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="noneAppend">
           <property name="text">
            <string>&amp;None</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" colspan="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>(Supported formats are: BMP,JPG,PNG,PBM,PGM,PPM,XBM,XPM)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_10">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default location for saving PDF files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>     PDF Save Location: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="savePdfDir">
          <property name="mode">
           <set>KFile::Directory</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Directory used for temporary files. Any existing local folder with read-write access will do. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Temp Files Location:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Standard Mode:&lt;/span&gt; All functions will be available. User interaction required.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Fast Image Mode: &lt;/span&gt;No user interaction. A unique identifier will be automatically appended to &lt;span style=&quot; font-style:italic;&quot;&gt;Image Name Prefix. &lt;/span&gt;Output images will be saved in the &lt;span style=&quot; font-style:italic;&quot;&gt;Image Save Location&lt;/span&gt; defined below. &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Fast PDF Mode: &lt;/span&gt;No user interaction. A unique identifier will be automatically appended to &lt;span style=&quot; font-style:italic;&quot;&gt;PDF Name Prefix. &lt;/span&gt;Output images will be saved in the &lt;span style=&quot; font-style:italic;&quot;&gt;PDF Save Location&lt;/span&gt; defined below.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Save &amp;mode:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>saveMode</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="saveMode">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Standard Mode:&lt;/span&gt; All functions will be available. User interaction required.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Fast Image Mode: &lt;/span&gt;No user interaction. A unique identifier will be automatically appended to &lt;span style=&quot; font-style:italic;&quot;&gt;Image Name Prefix. &lt;/span&gt;Output images will be saved in the &lt;span style=&quot; font-style:italic;&quot;&gt;Image Save Location&lt;/span&gt; defined below. If not used with caution, this may overwrite existing files!&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Fast PDF Mode: &lt;/span&gt;No user interaction. A unique identifier will be automatically appended to &lt;span style=&quot; font-style:italic;&quot;&gt;PDF Name Prefix. &lt;/span&gt;Output PDF files will be saved in the &lt;span style=&quot; font-style:italic;&quot;&gt;PDF Save Location&lt;/span&gt; defined below. If not used with caution, this may overwrite existing files!&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Save to Single PDF: &lt;/span&gt;Use this mode to combine several documents into a single PDF file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="currentText">
         <string>Save to Single PDF</string>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Standard Mode</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fast Save to Image</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fast Save to PDF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Save to Single PDF</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KUrlRequester" name="saveImgDir">
        <property name="mode">
         <set>KFile::Directory</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default location for saving images.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>   Image Save Location:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="2">
       <widget class="QLineEdit" name="tmpDir">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Directory used for temporary files. Any existing local folder with read-write access will do. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>/tmp</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_9">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For the image formats that support this option, quality usually defines the compression level when saving. Higher quality means less compression which, in turn, leads to bigger file size.&lt;/p&gt;&lt;p&gt;A value of 90 is considered to be a good choice.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Image Quality (1-100): </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="imgQuality">
          <property name="prefix">
           <string/>
          </property>
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_11">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default filename prefix when saving a documnet as PDF file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>        PDF Name Prefix: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pdfPrefix">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This entry defines the default filename prefix when saving a documnet as PDF file.&lt;/p&gt;&lt;p&gt;Notice that non unicode or special characters may cause errors.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>My PDF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default filename prefix when saving a documnet as image.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>      Image Name Prefix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="imgPrefix">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This  entry defines the default filename prefix when saving a documnet as image.&lt;/p&gt;&lt;p&gt;Notice that non unicode or special characters may cause errors.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>My Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default image format when saving a documnet.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Image Format: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="imgFormat">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This entry defines the image format to be used when saving. This is case insensitive, e.g.,  jpg,JPG and Jpg are all valid entries. &lt;/p&gt;&lt;p&gt;This also defines the filename's extension. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>jpg</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This affects the reuslting file size of the PDF file.&lt;/p&gt;&lt;p&gt;Higher quality means larger file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>                    PDF quality:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="pdfQuality">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This affects the reuslting file size of the PDF file.&lt;/p&gt;&lt;p&gt;Higher quality means larger file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>very low</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>low</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>high</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>very high</string>
           </property>
          </item>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QCheckBox" name="pdfEncrypt">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable this if you want to encrypt your file.&lt;/p&gt;&lt;p&gt;You'll be prompted to enter a password when the PDF file is ready.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Encrypt</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>setPreviewDPI</sender>
   <signal>toggled(bool)</signal>
   <receiver>previewDPI</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberAppend</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileNumber</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
